main.py

```diff
@@ -1,7 +1,44 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+
+    # Create
+    genres = [
+        "Western",
+        "Action",
+        "Dramma",
+    ]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    actors = [
+        ("George", "Clooney"),
+        ("Keanu", "Reeves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
+    for first_name, last_name in actors:
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+
+    # Update
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    (Actor.objects.filter(first_name="George", last_name="Klooney")
+     .update(last_name="Clooney"))
```

Comment:
Instead of enveloping this expression inside of `()`, remove them and make line breaking after `(`, so that new line would not start with `.update(...`